### ISSUE TITLE
Impossible to mark anything as watched or favorite

### ISSUE THREAD

**Issue by ieure** [jellyfin/jellyfin-media-player#246]
**Describe the bug**

There's something wrong with how JFMP sends watched / faved status:

 - After watching a TV episode, even all the way to the end of the media file, it doesn't get marked as watched.
 - if I click the checkmark icon to manually mark it as watched, it turns red as if it has been watched, but the server status never updates.  Refreshing the JFMP view, or looking on another client shows it unwatched.
 - Clicking the heart icon does nothing.  The icon doesn't turn red, nor does the server reflect that it was faved.

The web and Android clients don't exhibit this behavior.

**To Reproduce**
1. Click the check or heart icon for a show.
2. Refresh the view.
3. Note that the status doesn't persist.

**Expected behavior**
The watched/faved icons should work when I click them.

Watching a show until the end (or close to the end) should mark it as watched.

**Desktop (please complete the following information):**
 - OS: Debian Bullseye
 - Version: JFMP 1.6.1, Jellyfin 10.7.7
 - Installation Method: Flatpak

**Comment by ieure** [comment]
Noting also that watch progress behaves similarly.

**Comment by bialabs** [comment]
same bug on windows, have you found a solution?

**Comment by ieure** [comment]
> same bug on windows, have you found a solution?

Yes, I stopped using this software.  Now using the web interface with jellyfin-mpv-shim.

**Comment by sevenrats** [comment]
I'm not sure what to do with this bug report. These features work on every platform we have tested. If you think of anything more specific please add it.

**Comment by sevenrats** [comment]
woops accidental close sorry.

**Comment by ieure** [comment]
Is there some kind of troubleshooting/diagnostic procedure we could go through to help you reproduce it?  Are there particular logs that would be valuable to attach?

The bug most definitely exists, multiple people have encountered it, so the next step is to get it reproduceable so it can be fixed.

**Comment by andrewrabert** [comment]
This Qt desktop client has been deprecated in favor of a rewrite: https://github.com/jellyfin/jellyfin-desktop
